src/vmm/src/device_manager/persist.rs

```diff
@@ -573,9 +585,12 @@ impl<'a> Persist<'a> for MMIODeviceManager {
         }
 
         for net_state in &state.net_devices {
+            let interrupt =
+                Arc::new(IrqTrigger::new().expect("Could not create interrupt for MMIO device"));
```

Comment:
I saw comment from @roypat above about removing result from an `IrqTrigger::new`. Did you decide not to do it? Or maybe do in a separate PR?